rawx/chunk_info.go

```diff
@@ -191,9 +188,14 @@ func (chunk *chunkInfo) loadFullPath(inChunk fileReader, chunkID string) error {
 }
 
 func (chunk *chunkInfo) loadAttr(inChunk fileReader, chunkID string) error {
+	buf := make([]byte, 1024, 1024)
```

Comment:
It seems that using a snipet like this:
```
buf := make([]byte, 0, 32)
sz, err := syscall.Getattr(path, "user.plop", buf)
if err != nil { panic("arg"); }
log.Println(buf[:sz])
```
... it will always display an array of zeroes. Even if the low-level unsafe calls use the allocated of the buffer instead of its element count, getting a slice will implicitly init the elements to 0.